package.json

```diff
@@ -45,67 +45,66 @@
   "devDependencies": {
     "#": "link:./src/filters",
     "@11ty/eleventy": "^0.12.1",
-    "@11ty/eleventy-img": "^1.0.0-0",
+    "@11ty/eleventy-img": "^1.0.0",
     "@11ty/eleventy-plugin-rss": "^1.1.2",
     "@11ty/eleventy-plugin-syntaxhighlight": "^3.1.3",
-    "@babel/core": "^7.15.8",
-    "@babel/preset-env": "^7.15.8",
+    "@babel/core": "^7.16.0",
+    "@babel/preset-env": "^7.16.4",
     "@rollup/plugin-babel": "^5.3.0",
-    "@rollup/plugin-commonjs": "^21.0.0",
-    "@rollup/plugin-node-resolve": "^13.0.5",
+    "@rollup/plugin-commonjs": "^21.0.1",
+    "@rollup/plugin-node-resolve": "^13.0.6",
     "@tommoor/remove-markdown": "^0.3.2",
     "@types/markdown-it": "^12.2.3",
-    "@typescript-eslint/parser": "^4.33.0",
-    "@yarnpkg/sdks": "^2.4.2-0",
+    "@typescript-eslint/parser": "^5.6.0",
+    "@yarnpkg/sdks": "^2.5.1-rc.5",
     "accoutrement": "^3.0.1",
```

Comment:
@mirisuzanne I think we could also upgrade Accoutrement (and Sass) in this PR?

Reply:
yes please 👍🏼

Reply:
Doesn't need to hold up this PR.